lib/settings/pages/debug_settings_page.dart

```diff
@@ -244,19 +244,19 @@ class _DebugSettingsPageState extends State<DebugSettingsPage> {
                   contentText: l10n.deleteLocalDatabaseDescription,
                   onSecondaryButtonPressed: (dialogContext) => Navigator.of(dialogContext).pop(),
                   secondaryButtonText: l10n.cancel,
-                  onPrimaryButtonPressed: (dialogContext, _) async {
-                    String path = join(await getDatabasesPath(), 'thunder.db');
+                  // onPrimaryButtonPressed: (dialogContext, _) async {
```

Comment:
I think we need to figure out a way to handle this.
I don't see a drift method to delete the whole database.